site/docs/4.1/utilities/stretched-link.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: docs
+title: Stretched link
+description: Easily make an element which contains a stretched link clickable.
+group: utilities
+---
+
+Adding the `.stretched-link` class to a link will make the [containing block](https://developer.mozilla.org/en-US/docs/Web/CSS/Containing_block) of the `::after` pseudo element of the link clickable. In most cases, this means that an element with `position: relative;` which contains a link with the `.stretched-link` class is clickable.
+
+Cards are relative by default, so in this case you can safely add the `.stretched-link` class to a link in the card.
+
+{% capture example %}
+<div class="card" style="width: 18rem;">
+  {% include icons/placeholder.svg width="100%" height="180" class="card-img-top" text=" " title="Card image cap" %}
+  <div class="card-body">
+    <h5 class="card-title">Card with stretched link</h5>
+    <p class="card-text">Some quick example text to build on the card title and make up the bulk of the card's content.</p>
+    <a href="#" class="btn btn-primary stretched-link">Go somewhere</a>
+  </div>
+</div>
+{% endcapture %}
+{% include example.html content=example %}
+
+Media objects are not relative by default, we need to add the `.position-relative` here to prevent the link from stretching outside the media object.
```

Comment:
```suggestion
Media objects do not have `position: relative` by default, so we need to add the `.position-relative` here to prevent the link from stretching outside the media object.
```